scripts/src/main/resources/scripts/command/gcviewer

```diff
@@ -16,9 +16,16 @@ GCVIEWER_HOME="${DEVON_IDE_HOME}/software/gcviewer"
 TOOL_VERSION_COMMAND="-"
 # shellcheck source=scripts/commandlet-cli
 source "$(dirname "${0}")"/../commandlet-cli
+
 # gcviewer installation
 function doSetup() {
   doDevonCommand java setup silent
+  local current_version
+  current_version=$(doDevonCommand gcviewer version get | sed -n '2 p')
+  if [[ "${GCVIEWER_VERSION}" != "${current_version}" ]]
```

Comment:
avoid double brackets that may not work on MacOS due to old bash version.
Further consider the following cases:
* `GCVIEWER_VERSION` may not be defined.
* `GCVIEWER_VERSION` could be set to a version pattern (#893)

In both cases the existing installation would be deleted and the same version would be downloaded again.
However, after all this may still be the easiest fix as we otherwise would have to resolve the version to be installed duplicating some logic that is already done inside `doInstall`.

A smarter approach could also be to add a tweak to the if in line 32 that checks if there are multiple gcviewer JAR files now in `${GCVIEWER_HOME}` and delete all but the latest one (according to mtime - in case the mtime is pointing to the date+time when the file was downloaded/copied).

Reply:
i implemented this suggestion in the latest commit